src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private static final int NUMBER_THREE = 3;
+    private static final int NUMBER_FOUR = 4;
```

Comment:
I dont think we need to move these numbers to constants, as we use them in formula that wont change(fix in all Figure child classes)